catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergConfig.java

```diff
@@ -69,6 +71,28 @@ public class IcebergConfig extends Config {
           .booleanConf()
           .createWithDefault(true);
 
+  public static final ConfigEntry<String> ICEBERG_METRICS_STORAGE =
+      new ConfigBuilder(IcebergMetricsManager.ICEBERG_METRICS_STORAGE)
+          .doc("The storage to store Iceberg metrics")
+          .version("0.4.0")
+          .stringConf()
+          .createWithDefault(MemoryMetricsStorage.ICEBERG_METRICS_STORAGE_MEMORY_NAME);
```

Comment:
I think you should not have a default in-memory metrics store, it's not a recommended store to store metrics. Instead I would choose to use a blackhole-like store to throw metrics directly as default.

Reply:
done